packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -399,14 +399,33 @@ RouterService.reopen(Evented, {
   currentURL: readOnly('_router.currentURL'),
 
   /**
-    The `location` property determines the type of URL's that your
+    The `location` property determines the type of URLs that your
     application will use.
+
     The following location types are currently available:
     * `auto`
     * `hash`
     * `history`
     * `none`
 
+    You can pass a location type to force a particular `location` API
+    implementation to be used in your application. For example, to set
+    the `history` type:
+
+    ```config/environment.js
+    'use strict';
```

Comment:
There are some indentation issues.
Can you make sure these lines through to the end of the code block are indented in two spaces?